.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+tasks:
+  - name: Nats
+    init: |
+      docker-compose pull
+    command: |
+      docker-compose up -d
+  - name: PWA
+    init: |
+      cd app/
+      pnpm install
```

Comment:
> Your pnpm version is incompatible with "/workspace/kiel-live/app".
> 
> Expected version: 6
> Got: 7.1.2

Reply:
Not again ...

Reply:
use your dockerfile approach to install pnpm6